esp_serial_slave_link/README.md

```diff
@@ -0,0 +1,9 @@
+# Espressif Serial Slave Link (ESSL) component
+
+This component used to reside in [esp-idf](https://github.com/espressif/esp-idf) project as its component.
+
+It's used on the HOST, to communicate with ESP chips as SLAVE via SDIO/SPI slave HD mode.
+
+The port layer (essl_sdio/essl_spi.c) are currently only written to run on ESP masters, but you may also modify them to work on more platforms.
+
+See more documentation: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/protocols/esp_serial_slave_link.html
```

Comment:
Do we plan to maintain documentation of this component on IDF side? I think this might be little inconsistent as any change here can break the documentation. Can we consider moving documentation to component here?

Reply:
Not sure which one is better actually. I would suggest to move the part of documentation here after this repo has some documentation system

Reply:
@igrr What would you recommend here?